app/helpers/pagination_helper.rb

```diff
@@ -49,7 +49,7 @@ def pages(page, total_pages) # rubocop:disable Metrics/AbcSize, Metrics/Cyclomat
           nil
         elsif (page < 5 && i == 6) ||
               (page > total_pages - 4 && i == total_pages - 5) ||
-              (page > 4 && page < total_pages - 3 && (i == page + 2 || i == page - 2))
+              (page > 4 && page < total_pages - 3 && [page + 2, page - 2].include?(i))
```

Comment:
I can't decide if this is easier to read. Well done chipping away at that file. It's a huge logic pile.